templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+                            Lauren Lum
+                        <p>
+                    </div>
+                </div>
+                <div class="container carousel-entry has-background-image" style='display: flex; justify-content: space-between; background-image: url({{ static_path "bmviii_testimonial1_background.svg" }})'>
+                    <div class="column is-half">
+                        <img alt="Samuel Ingram" src={{ static_path "sam.jpg" }}>
+                    </div>
+                    <div class="column is-half" style="word-wrap: break-word">
+                        <p>
+                            BoilerMake gave me the resources and support I needed to create a project I was truly proud of.
+                            If you have an idea and a passion for creating something from scratch, don’t let this opportunity pass you up!
+                        </p>
+                        <p style="font-family: Cabin; family-style: normal; font-weight: 600; font-size: 20px" >
+                            Samuel Ingram
+                        <p>
+                    </div>
+                </div>
+                <div class="container carousel-entry has-background-image" style='display: flex; background-image: url({{ static_path "bmviii_testimonial1_background.svg" }});  justify-content: space-between'>
+                    <div class="column is-half">
+                        <img alt="Kevin Cardona" src={{ static_path "kevin.jpg" }}>
+                    </div>
+                    <div class="column is-half" style="word-wrap: break-word">
+                        <p>
+                            BoilerMake gave me the resources and support I needed to create a project I was truly proud of.
+                            If you have an idea and a passion for creating something from scratch, don’t let this opportunity pass you up!
+                        </p>
+                        <p style="font-family: Cabin; family-style: normal; font-weight: 600; font-size: 20px" >
+                            Kevin Cardona
+                        <p>
+                    </div>
+                </div>
+                <div class="carousel-button-next"></div>
+                <div class="carousel-button-prev"></div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="section has-background-image" id="sponsors" style='background-image: url({{ static_path "bmviii_sponsors_background.svg" }})'>
+    <div class="container">
+        <div class="columns is-centered content">
+            <div class="column is-2 bmviii-section-header" style="margin-left: 48px; margin-right: 12px; margin-top:72px">
+                <img alt="FAQ" src={{ static_path "bmviii_sponsors_text.svg" }}>
+            </div>
+        </div>
+    </div>
+    <div class="container" style="display: flex; justify-content: space-between; margin-top: 90px">
```

Comment:
oof
![Screen Shot 2020-10-05 at 21 26 07](https://user-images.githubusercontent.com/10781907/95148472-01415d00-0749-11eb-81b3-b62126d6ca26.png)

I know the design just has the logos on top of the water background but that never works out in practice, we just can't control what their logos will look like so we the only way we can make sure that things look somewhat normal is to put the logos on just like a plain white background.  You can see how we did that in the BM7 site.

Reply:
ofc you'll still have the water background around that plain white thing that holds the logos.

Reply:
Yeah I had that in as like a placeholder, but I think design is gonna send over images to use